src/podman-desktop.css

```diff
@@ -0,0 +1,483 @@
+@scope (.light) {
```

Comment:
not a blocker, jsut a remark: I am wondering if we should not publish this somewhere, as we will always need to update it to keep it updated :(

Reply:
yes, it's my intent 👍